src/Avro/Client.php

```diff
@@ -16,12 +17,18 @@ class Client
      */
     private $client;
 
-    public function __construct(string $url)
+    /**
+     * @var array
+     */
+    private $options;
+
+    public function __construct(array $options)
     {
         // Construct is temporary as we will
         // put everything on the service provider.
-        $this->baseUrl = $url;
-        $this->client = app(GuzzleHttp::class, ['config' => ['timeout' => 40000]]);
+        $this->baseUrl = $options['url'];
```

Comment:
should these options be validated? there is no doc saying what is needed to configure the client